### PR TITLE
Adding post 'Nine Techniques to Make Conference Talks Suck Less'

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

File: content/blog/nine-techniques-make-conference-talks-suck-less.asciidoc [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+The reason being that the high pace also gives me some energy and flow which I hope becomes apparent to the audience.
+I believe viewers (and I) are better off with me doing a passionate talk which may be a bit too fast,
+instead of one which has a slower pace but lacks the right amount of energy.
+
+I think that's generally applicable:
+You don't like talking about concepts, but love showing how things work in action?
+Then shorten the former and make more room for a live demo.
+You enjoy discussing live questions? Make more time for the Q&A.
+This all is to say, rather than excessively focussing on things you perceive as your weak sides,
+rather leverage your strong suites.
```
Comment (hpgrahsl) [inline content/blog/nine-techniques-make-conference-talks-suck-less.asciidoc:216]: Rather 2x in same sentence. Rephrase?

**Review by hpgrahsl — APPROVED** [review]
Nice one. Just a few nit picks I commented ;)

File: content/blog/nine-techniques-make-conference-talks-suck-less.asciidoc [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+i.e. its key to motivate why and how it matters to them,
+and how learning about the solution will benefit them.
+
+Another approach is to present the key learnings (for instance three, see below) which the audience will make during the talk.
+While this may sound similar to an agenda slide,
+the framing is different: it's taking the perspective of the listener and what's in it for them by sticking through your session.
+Don't lead with your personal introduction;
+if you're known in the field, people don't care.
+And if you're not, well, they probably also won't care.
+In any case, telling much about yourself is what will people attract to your talk.
```
[the remark this answers is not in the document]
Comment (gunnarmorling) [reply content/blog/nine-techniques-make-conference-talks-suck-less.asciidoc:67]: Ha, good spot, there's a "not" missing. Meant to say "Is _not_ what will...", i.e. exactly the opposite. Thanks for catching this!

File: content/blog/nine-techniques-make-conference-talks-suck-less.asciidoc [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+I.e. it should form a cohesive unit, rather than being a collection of random unrelated talking points.
+
+Other story telling techniques can be employed to great effect as well,
+such as doing a quick wrap-up when finishing a key section of your talk,
+or adding little "side quests" for things you really want to mention but which are not strictly related to the main storyline.
+
+In terms of crafting a story,
+I try to start early and collect input over a longer period of time,
+using a mind map for instance.
+This allows you to wholistically discuss the given topic,
```
[the remark this answers is not in the document]
Comment (gunnarmorling) [reply content/blog/nine-techniques-make-conference-talks-suck-less.asciidoc:96]: Yeah, good point. I need to rephrase this. The idea is that you have a wide range of _potential_  things from which you can choose. So what you end up discussing is the most relevant stuff. Does that make sense?

File: content/blog/nine-techniques-make-conference-talks-suck-less.asciidoc [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+One example:
+I tend to speak fast, _very_ fast, during talks.
+I'm well aware of that, listeners told me, a coach told me, I saw it myself in recordings.
+But it's somehow impossible for me to change it.
+If I really force myself hard to speak slower, it will work for a while,
+but typically I'll be back to my usual speed after a while.
+
+So I've decided to not fight against this any longer and just live with it.
+The reason being that the high pace also gives me some energy and flow which I hope becomes apparent to the audience.
+I believe viewers (and I) are better off with me doing a passionate talk which may be a bit too fast,
```
[the remark this answers is not in the document]
Comment (gunnarmorling) [reply content/blog/nine-techniques-make-conference-talks-suck-less.asciidoc:208]: I don't quite get what you say. In my case, high pace equates to more passion (I believe), which is what I'm saying here. What do you think should be changed?

File: content/blog/nine-techniques-make-conference-talks-suck-less.asciidoc (hunk 1 above)
Comment (gunnarmorling) [reply content/blog/nine-techniques-make-conference-talks-suck-less.asciidoc:216]: Indeed.

File: content/blog/nine-techniques-make-conference-talks-suck-less.asciidoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (hpgrahsl) [reply content/blog/nine-techniques-make-conference-talks-suck-less.asciidoc:96]: absolutely yes!

File: content/blog/nine-techniques-make-conference-talks-suck-less.asciidoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (hpgrahsl) [reply content/blog/nine-techniques-make-conference-talks-suck-less.asciidoc:208]: talking fast != more passion. fair enough if it feels like that for yourself but it's in general not true for the average speaker out there, at least not in my experience from listening to other talks. passion is either there or not - irrespective of speed of words or fast pace as you phrased it. but ok to disagree on that ;-)

**Pull request merged**